### PR TITLE
Updated Packages

### PR DESCRIPTION
**Pull request by Cartisim**

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -16,8 +16,8 @@ let package = Package(
             targets: ["Dribble"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/apple/swift-nio.git", from: "2.0.0"),
-        .package(url: "https://github.com/apple/swift-crypto.git", from: "1.0.0"),
+        .package(url: "https://github.com/apple/swift-nio.git", from: "2.41.1"),
+        .package(url: "https://github.com/apple/swift-crypto.git", from: "2.1.0"),
```
Comment (Joannis) [inline Package.swift]: I cannot do this, but we can specify `1.0.0 ..< 3.0.0`

**Comment by Cartisim** [comment]
OK, I update the swift-crypto range as you specified.

**Comment by Cartisim** [comment]
@Joannis Hey can we do something to move this one along?

**Comment by Joannis** [comment]
Sorry, I overlooked that it was still open

**Comment by Cartisim** [comment]
No sweat, thanks a million.

**Pull request merged**